src/EditorFeatures/Test2/GoToDefinition/GoToDefinitionTests.vb

```diff
@@ -960,54 +968,72 @@ class C
         End Sub
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.GoToDefinition)>
-        Public Sub TestCSharpGoToUnmanaged_Keyword()
+        Public Sub TestCSharpGoToOnBreakInSwitchStatement()
             Dim workspace =
 <Workspace>
     <Project Language="C#" CommonReferences="true">
         <Document>
-            class C&lt;T&gt; where T : un$$managed
```

Comment:
why were existing tests changed?

Reply:
if they werren't, can you please move your tests to the end to make the diff clear?

Reply:
The tests didn't change. Moved the tests